src/class/FileManager/FileManager.hpp

```diff
@@ -3,15 +3,24 @@
 #define FILE_MANAGER_HPP
 
 #include <cstdio>
+#include <dirent.h>
+#include <fstream>
+#include <iomanip>
 #include <iostream>
+#include <sstream>
 #include <string>
 #include <sys/stat.h>
 
+#include <converter/converter.hpp>
+
 #ifdef TEST_ON
 #define private public
 #define protected public
 #endif
 
+static const std::string PREVIOUS_DIR = "/..";
```

Comment:
these should be part of the class.
e.g. on line 48 declare `static const std::string PREVIOUS_DIR;`
and in FileManager.**cpp** write `const std::string FileManager::PREVIOUS_DIR("/..");`

when you write it like this, it is like the C static for specific file